src/powfacpy/results_interface.py

```diff
@@ -38,10 +38,10 @@ def export_to_pandas(self, result_objects, elements, variables):
         variables: List of variable names for each result variable
     """
     FILE_NAME = 'temp'
-    FILE_PATH = getcwd()
-    FULL_PATH = FILE_PATH + "\\" + FILE_NAME + ".csv"
+    file_path = getcwd()
```

Comment:
file_path and full_path are still constants and could be written in capital letters?